multinet/urls.py

```diff
@@ -40,7 +47,7 @@
     path('accounts/', include('allauth.urls')),
     path('oauth/', include('oauth2_provider.urls', namespace='oauth2_provider')),
     path('admin/', admin.site.urls),
-    path('api/upload/', include('s3_file_field.urls')),
+    path('api/s3-upload/', include('s3_file_field.urls')),
```

Comment:
Is there a good reason to change the name of the route?

Reply:
I didn't want there to be any confusion/conflict with the multinet specific `upload` endpoints